source/util/FileUtils.cpp

```diff
@@ -66,7 +66,13 @@ string FileUtils::ExtractExpandedPath(const string &filePath)
         return "";
     }
     wordexp_t word;
-    wordexp(filePath.c_str(), &word, 0); // TODO: How to handle error?
+    int status = wordexp(filePath.c_str(), &word, 0); // TODO: How to handle error?
+    if (status)
+    {
+        throw Aws::Iot::DeviceClient::Util::wordexp_fail_error(
```

Comment:
Minor: You prolly don't need the full namespace now. If you still see the error create a shorter synonym with a using declaration. https://en.cppreference.com/w/cpp/language/using_declaration